### ISSUE TITLE
X data and using content id

### ISSUE THREAD

**Issue by poi33** [enonic/lib-guillotine#248]
If i define my Xdata with an image selector how can I get its image url?

```
{ guillotine {
    getSite {
         x {
             name
             data
         }
     }
}}
```

Will return: 
```
getSite": {
        "x": [
          {
            "name": "x-data",
            "data": "{icon=<id_here>}"
          }
        ]
      }
```

So I will only get an ID and i never get to the image type too call its image Url

Same with xAsJson

```
"xAsJson": {
          "<app-name>": {
            "page-data": {
              "icon": "<id_here>",
            }
          }
        },
```

I never get to the image type that supports imageUrl

**Comment by anatol-sialitski** [comment]
@poi33 could please check it on v6.0.0?

**Comment by poi33** [comment]
This seems to be fixed in 6.0.0-b3 👍
